openfisca_uk/variables/demographic/person/employment.py

```diff
@@ -0,0 +1,71 @@
+from openfisca_uk.model_api import *
+
+
+class EmploymentStatus(Enum):
+    FT_EMPLOYED = "Full-time employed"
+    PT_EMPLOYED = "Part-time employed"
+    FT_SELF_EMPLOYED = "Full-time self-employed"
+    PT_SELF_EMPLOYED = "Part-time self-employed"
+    UNEMPLOYED = "Unemployed"
+    RETIRED = "Retired"
+    STUDENT = "Student"
+    CARER = "Carer"
+    LONG_TERM_DISABLED = "Long-term sick/disabled"
+    SHORT_TERM_DISABLED = "Short-term sick/disabled"
+    OTHER_INACTIVE = "Inactive for another reason"
+    CHILD = "Child"
+
+
+class employment_status(Variable):
+    value_type = Enum
+    entity = Person
+    possible_values = EmploymentStatus
+    default_value = EmploymentStatus.UNEMPLOYED
+    label = "Employment status"
+    documentation = "The employment status of this person"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        PT_HOURS_THRESHOLD = 16
```

Comment:
Could you make this a parameter? What programs does it affect?

Reply:
This doesn't affect programs, but is more of a general descriptor - e.g. if we broke down a result by employment status. Not sure about parametrising a non-policy parameter